misc/json-to-record-converter/README.md

```diff
@@ -0,0 +1,530 @@
+# JSON to Record Converter
+
+The tool is intended to map JSON values directly to Ballerina records without converting the JSON value to its schema. The tool individually analyzes the JSON value nodes and maps it to the Ballerina record either in-line or other. The JSON value is parsed as a parse tree using Google’s gson package. And the tree is traversed to generate the Ballerina record.
+
+## Features
+In the current implementation we support JSON value to Ballerina Record Conversion
+
+##### _Example JSON Value_
```

Comment:
```suggestion

### Example
**The JSON value**
```
Why is this heading level 5? We can add a heading as `Example` and then have sub-topics for the `JSON value` and the `Converted Ballerina record`.